src/app/DTT/upload/Snap.js

```diff
@@ -1,43 +1,61 @@
 import React, { useRef, useCallback } from 'react'
 import { useUserMedia } from 'utils/hooks'
-import { Button, VStack, HStack } from '@chakra-ui/react'
+import { Button, VStack, HStack, useToast, IconButton } from '@chakra-ui/react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTimes } from '@fortawesome/free-solid-svg-icons'
 
 const CAPTURE_OPTIONS = {
   video: true,
 }
 
 const Snapshot = ({ handleSave, setLoading, setOpenSnap }) => {
+  const toast = useToast()
   const videoRef = useRef()
-  const stream = useUserMedia(CAPTURE_OPTIONS)
-
-  if (stream && videoRef.current && !videoRef.current.srcObject) {
-    videoRef.current.srcObject = stream
-  }
-
-  const onCanPlay = () => videoRef.current.play()
+  const stream = useUserMedia(CAPTURE_OPTIONS, err =>
+    toast({
+      title: 'Oops',
```

Comment:
"Oops, it appears that I have made a human mistake, as humans do, ha ha ha ha!!!1101001" - 🤖

Reply:
haha